CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 * Add a null check for `Severity` to the notify override method. [#214](https://github.com/bugsnag/bugsnag-java/pull/214)
 
+* Add handling for non serializable object passed to metadata. [#219](https://github.com/bugsnag/bugsnag-java/pull/219)
```

Comment:
This and the PR description need updating for the amended approach - we're not handling non-serializable objects directly: just providing a way for people to do so if they provide their own serializer.

Reply:
The PR title and description are still out of date